Main.java

```diff
@@ -0,0 +1,83 @@
+package com.company;
+
+import java.util.*;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+
+        int[] arr1 = new int[]{1, 2, 3, 4, 5};
+        int[] arr2 = new int[]{2, 3, 4, 6, 10};
+        int[] arr3 = mergeArray(arr1, arr2);
+
+        Arrays.stream(arr3).forEach((i) -> System.out.print(arr3[i - 1] + " "));
+        System.out.println();
+
+        String[] roles = new String[]{"Городничий",
+                "Аммос Федорович",
+                "Артемий Филиппович",
+                "Лука Лукич"};
+
+        String[] textLines
+                = new String[]{"Городничий: Я пригласил вас, господа, с тем, чтобы сообщить вам пренеприятное известие: к нам едет ревизор.",
+                "Аммос Федорович: Как ревизор?",
+                "Артемий Филиппович: Как ревизор?",
+                "Городничий: Ревизор из Петербурга, инкогнито. И еще с секретным предписаньем.",
+                "Аммос Федорович: Вот те на!",
+                "Артемий Филиппович: Вот не было заботы, так подай!",
+                "Лука Лукич: Господи боже! еще и с секретным предписаньем!"
+        };
+
+        long start = System.nanoTime();
+        System.out.println(printTextPerRole(roles, textLines));
+        long finish = System.nanoTime();
+        System.out.println(finish - start);
+
+    }
+
+    public static int[] mergeArray(int[] arr1, int[] arr2) {
+        int[] arr = new int[arr1.length + arr2.length];
+        int index_arr1 = 0;

```

Comment:
https://google.github.io/styleguide/javaguide.html#s5.2.5-non-constant-field-names
Non-constant field names (static or otherwise) are written in lowerCamelCase.